data/report_templates/project_summary.md

```diff
@@ -154,6 +158,10 @@ NGI ID | User ID | Status
 
 In cases where samples and/or libraries have failed the QC, the user is always consulted regarding how to proceed. If the user wishes to proceed to sequence the failed samples and/or libraries, NGI bears no responsibility regarding the quality and number of reads of the sequenced sample data.
 
+## User-provided samples, information and data
+
+The NGI does not take responsibility for how user-provided samples, information, or data may affect sequencing results.
```

Comment:
Should it be 'The' NGI or just NGI? The rest of the document doesn't use the definite version.

Reply:
I asked Phil to look at it, and he specifically wanted it to say "the" NGI :-D
Hm so either I would have to change the whole document or we remove the "the", maybe it's the easiest to just remove the "the", sorry Phil :-D

Reply:
Remove it, I have been outvoted on this many times. I will keep fighting, but I have lost this battle ☝🏻 😆